ctam/interfaces/fw_update_ifc.py

```diff
@@ -444,7 +454,7 @@ def RedFishFWUpdate(self, BinPath, URI, targets=[], is_multipart=False):
             headers = {"Content-Type": "multipart/form-data"}
             body = {
                 "UpdateFile": (BinPath, open(BinPath, "rb"), "application/octet-stream"),
-                "UpdateParameters" : ("Targets", json.dumps({"Targets": targets, "ForceUpdate": True if self.dut().multipart_force_update else False}),'application/json')
+                "UpdateParameters" : ("Targets", json.dumps({"Targets": targets, "ForceUpdate": True if is_force_update else False}),'application/json')
```

Comment:
Are we saying that downgrade requires a ForceUpdate? Then we should this ask this as a standard?

Reply:
Yes for Multipart Update we need ForceUpdate for downgrade. We have the ForceUpdate under UpdateParameters in the redfish schema: https://redfish.dmtf.org/schemas/v1/UpdateService.v1_14_1.json
For update using HttpPushUri, ForceUpdate is not needed.